src/ui/mainwindow.cpp

```diff
@@ -324,95 +323,24 @@ void MainWindow::loadIcons()
-
-
-    QLabel *label;
-    QMargins tmpMargins;
-
-    label = new ClickableLabel("File Format", this);
-    label->setSizePolicy(QSizePolicy::Minimum, QSizePolicy::Fixed);
-    label->setMinimumWidth(150);
-    tmpMargins = label->contentsMargins();
-    label->setContentsMargins(tmpMargins.left(), tmpMargins.top(), tmpMargins.right() + 10, tmpMargins.bottom());
-    layout->addWidget(label);
-    m_statusBar_fileFormat = label;
-    connect(dynamic_cast<ClickableLabel*>(label), &ClickableLabel::clicked, [this](){
-        ui->menu_Language->exec( QCursor::pos() );
-    });
-
-
-    label = new QLabel("Ln 0, col 0", this);
-    label->setSizePolicy(QSizePolicy::Minimum, QSizePolicy::Fixed);
-    label->setMinimumWidth(120);
-    tmpMargins = label->contentsMargins();
-    label->setContentsMargins(tmpMargins.left(), tmpMargins.top(), tmpMargins.right() + 10, tmpMargins.bottom());
-    layout->addWidget(label);
-    m_statusBar_curPos = label;
-
-    label = new QLabel("Sel 0, 0", this);
-    label->setSizePolicy(QSizePolicy::Minimum, QSizePolicy::Fixed);
-    label->setMinimumWidth(120);
-    tmpMargins = label->contentsMargins();
-    label->setContentsMargins(tmpMargins.left(), tmpMargins.top(), tmpMargins.right() + 10, tmpMargins.bottom());
-    layout->addWidget(label);
-    m_statusBar_selection = label;
-
-    label = new QLabel("0 chars, 0 lines", this);
-    label->setSizePolicy(QSizePolicy::MinimumExpanding, QSizePolicy::Fixed);
-    tmpMargins = label->contentsMargins();
-    label->setContentsMargins(tmpMargins.left(), tmpMargins.top(), tmpMargins.right() + 10, tmpMargins.bottom());
-    layout->addWidget(label);
-    m_statusBar_length_lines = label;
-
-    label = new QLabel("EOL", this);
-    label->setSizePolicy(QSizePolicy::Minimum, QSizePolicy::Fixed);
-    label->setMinimumWidth(118);
-    tmpMargins = label->contentsMargins();
-    label->setContentsMargins(tmpMargins.left(), tmpMargins.top(), tmpMargins.right() + 10, tmpMargins.bottom());
-    layout->addWidget(label);
-    m_statusBar_EOLstyle = label;
-
-    label = new ClickableLabel("Encoding", this);
-    label->setSizePolicy(QSizePolicy::Minimum, QSizePolicy::Fixed);
-    label->setMinimumWidth(118);
-    tmpMargins = label->contentsMargins();
-    label->setContentsMargins(tmpMargins.left(), tmpMargins.top(), tmpMargins.right() + 10, tmpMargins.bottom());
-    layout->addWidget(label);
-    m_statusBar_textFormat = label;
-    connect(dynamic_cast<ClickableLabel*>(label), &ClickableLabel::clicked, [this](){
-        ui->menu_Encoding->exec(QCursor::pos());
-    });
-
-    label = new QLabel(tr("INS"), this);
-    label->setSizePolicy(QSizePolicy::Minimum, QSizePolicy::Fixed);
-    label->setMinimumWidth(40);
-    layout->addWidget(label);
-    m_statusBar_overtypeNotify = label;
-
-
-    status->addWidget(scrollArea, 1);
-    scrollArea->setFixedHeight(frame->height());
+    m_sbDocumentInfoLabel = new QLabel;
+    m_sbDocumentInfoLabel->setMinimumWidth(1);
```

Comment:
Why set minimum width? It doesn't seem to make a difference.

Reply:
I don't mind removing it.  There's a couple of oddities related to not having a minimum size and working with tiny tiny windows.... but its really an edge case and I'm probably being paranoid.

Reply:
Actually... this one needs to stay.   The minimum size of the entire window is limited to the text in the widget there.  Assuming they start off with a small window and open a large document the window would automatically resize to fit the obviously larger text.

I'd rather leave it up to the user on cutting off informational text themselves.